src/components/containers/fieldset/Fieldset.js

```diff
@@ -40,7 +40,7 @@ function Fieldset({ additionalLegendClasses, legendText, children, theme }) {
 
 Fieldset.propTypes = {
   /** Determine whether or not to add a legend and what text to display  */
-  legendText: PropTypes.string,
+  legendText: PropTypes.object,
```

Comment:
I think I'm misunderstanding this. The pass in an object, but the `renderLegend` function is expecting text? Should we be expecting a certain shape of object for `legendText`?